### PR TITLE
Fix file input button shadow

### PR DESCRIPTION
**Pull request by llorca**
That shadow actually works fine locally without this change, but not on the docs website (the `::after` has no `box-shadow` rule). This PR explicitly adds a `box-shadow` rule to the `::after` without relying on the mixin

### CODE REVIEW

**Comment by llorca** [comment]
I messed up the first commit, but funny thing: it works with `$pt-input-box-shadow` but not `$button-box-shadow`

File: packages/core/src/components/button/_common.scss [hunk 1]
```diff
@@ -21,42 +21,32 @@ CSS `border` property issues:
...
+                            0 1px 1px rgba($black, 0.2) !default;
+$button-box-shadow-overlay-active: 0 0 0 $button-border-width rgba($black, 0.2),
+                                   inset 0 1px 1px rgba($black, 0.1) !default;
+
+$dark-button-box-shadow: 0 0 0 $button-border-width rgba($black, 0.4) !default;
+$dark-button-box-shadow-active: 0 0 0 $button-border-width rgba($black, 0.6),
+                                inset 0 1px 2px rgba($black, 0.2) !default;
+$dark-button-intent-box-shadow: 0 0 0 $button-border-width rgba($black, 0.4) !default;
+$dark-button-intent-box-shadow-active: 0 0 0 $button-border-width rgba($black, 0.4),
+                                       inset 0 1px 2px rgba($black, 0.2) !default;
```
Comment (giladgray) [inline packages/core/src/components/button/_common.scss]: revert these whitespace changes please, the previous syntax was more legible.

File: packages/core/src/components/forms/_file-input.scss [hunk 2]
```diff
@@ -129,3 +129,9 @@ $file-input-button-padding-large: ($pt-input-height-large - $pt-button-height) /
     }
   }
 }
+
+// hack to force the button shadow to display since
+// it doesn't render correct via the `pt-button` mixin
+// stylelint-disable-next-line
+.pt-file-upload-input::after { box-shadow: $button-box-shadow; }
+// stylelint-enable
```
Comment (giladgray) [inline packages/core/src/components/forms/_file-input.scss]: don't need this since you're using `-next-line` (automatic re-enable)

**Comment by llorca** [comment]
like shooting a target with eyes closed. can't believe this worked haha

**Pull request merged**